h2/src/main/org/h2/table/TableFilter.java

```diff
@@ -1076,7 +1076,7 @@ public void setDerivedColumns(ArrayList<String> derivedColumnNames) {
         if (count != derivedColumnNames.size()) {
             throw DbException.get(ErrorCode.COLUMN_COUNT_DOES_NOT_MATCH);
         }
-        HashMap<Column, String> map = new HashMap<>(count);
+        HashMap<Column, String> map = new HashMap<>();
```

Comment:
why are you removing some of them?

Reply:
To avoid code like `new HashMap<>((int) (count / 0.75f))` in places where default 12 elements (capacity 16) is enough in almost all cases.

Reply:
I can use such code instead, of course. Or multiply count by 2 for simplicity. I'm not sure about the best choice here.

Reply:
None of this stuff shows up on any of my profiles, so I don't care either way

Reply:
I checked all these places where I removed parameter once again. I think we can just use the default capacity in these places.